src/Packagist/WebBundle/Resources/public/font/config.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
I had to manually look up each icon that was used to create the custom font, which took a while, so I figured why not commit this so it's easier to make future changes?